### ISSUE TITLE
As an IO, while I'm awaiting my operator approval, I want to see a Request Pending page

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#395]
#### Acceptance Criteria:

Given I have requested access to an Operator and the status of my access request is Pending
When I click on the "My Operator" tile in my dashboard
Then I see a generic "Your Request is Pending" page

(No wireframe since it's a temporary solution. Dev gets to freestyle the design)

#### Development Checklist:

- [ ] create an endpoint to fetch user operator status (creating a new endpoint rather than refactoring an existing one so it'll be easier to clean up later once the real solution is in place)
- [ ] create a bare bones pending page
- [ ] conditionally link to the pending page if status is pending
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by BCerki** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-Moose-Sprint-Board-60d216175a2689000ec774d0/issues/bcgov/cas-registration/395?planning-poker) with Zenhub @BCerki @dleard @joshgamache @JoshLarouche @mikevespi @Sepehr-Sobhani @shon-button @marcellmueller

**Comment by BCerki** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-Moose-Sprint-Board-60d216175a2689000ec774d0/issues/bcgov/cas-registration/395?planning-poker) with Zenhub @andrea-williams

**Comment by dleard** [comment]
@BCerki @marcellmueller is this the same as or being iterated on in #441? Currently in develop, we are still being directed to the 'select-operator' page.
cc @hannavovk

**Comment by BCerki** [comment]
The pending page was a temporary solution because we won't have a read-only user operator form until #441 is in. A bug was introduced at some point that broke this page. I think @marcellmueller said it was just a matter of changing some capitalization to fix it.

**Comment by marcellmueller** [comment]
Closing since `APPROVED` and `PENDING` status get redirected to the form now.
